lib/api/pokedex_api_spec/model/pokemon_model.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'pokemon_model.freezed.dart';
+
+part 'pokemon_model.g.dart';
+
+@freezed
+class PokemonModel with _$PokemonModel {
+  factory PokemonModel({
+    @Default('') @JsonKey(name: 'name') String name,
```

Comment:
Is this your final model for your Pokemon object? 🤔 

If no => have a todo regarding important stuff to include here 🙂 
if yes => among all the properties that can be retrieved from the api, are you sure you only needed the `name` property? How about i.e. `id` and other properties...? 🙂

Reply:
yes this will be my final object for the Pokemon but I'm planning to create a seperate object for the types and such but I have also included the id and url of the pokemon because initially I thought that I will only need the name for the initialization